prodzilla.yml

```diff
@@ -1,40 +1,147 @@
 probes:
-  - name: your-post
-    url:	https://your.site/some/path
-    http_method: POST
-    with:
-      headers:
-        x-client-id: ClientId
-      body: '"{"test": true}"'
+  - name: api-core-health
+    url:	https://api.xylex.ai/systems/health/main
+    http_method: GET
     expectations:
       - field: StatusCode
         operation: Equals 
         value: "200"
     schedule:
       initial_delay: 0
-      interval: 60
+      interval: 15
     alerts:
-      - url: https://notify.me/some/path
+      - url: https://discord.com/api/webhooks/1262877118573052018/C9TRgwPFlj5swDaHmH5LUhQqryUYkJMRZdzSnrlLGkqVYr9XDPSL1R03EKpYwtP5YURE
```

Comment:
I think you accidentally leaked this webhook by the way, probably worth rotating it in your server 🙏 I'd recommend using environment variable substitution for this in the future so it isn't accidentally committed, I also tend to have a secondary YAML-file for fiddling with changes locally which I run with `-f path/to/file.yaml` 😄

Reply:
hahah bro you can ping my log channel i appreciate you looking out for me, but its okay